charms/jupyter-controller/config.yaml

```diff
@@ -5,5 +5,5 @@ options:
     description: Enables culling of idle Jupyter pods
   use-istio:
     type: boolean
-    default: false
+    default: true
     description: Manually enable Istio integration, instead of via relations
```

Comment:
Reminder of our discussion: add in the description of the config that is should be always true for kubeflow users